src/Sentry.Unity.Editor.iOS/SentryXcodeProject.cs

```diff
@@ -1,9 +1,8 @@
 using System;
 using System.IO;
 using System.Linq;
+using System.Reflection;
 using Sentry.Extensibility;
-using UnityEditor.iOS.Xcode;
-using UnityEditor.iOS.Xcode.Extensions;
 using UnityEngine;
 
 namespace Sentry.Unity.Editor.iOS
```

Comment:
Worth adding a note like:

// The decision to use reflection here came after many attempts to avoid it
// Allowing customers to export an xcode project outside a Mac while not
// requiring users to install the iOS tools on Windows and Linux became a challenge
// More context: https://github.com/getsentry/sentry-unity/pull/592 and https://github.com/getsentry/sentry-unity/issues/400